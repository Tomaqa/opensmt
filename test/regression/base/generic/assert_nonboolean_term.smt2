(set-logic QF_LRA)
(declare-fun x () Real)
(assert x)
