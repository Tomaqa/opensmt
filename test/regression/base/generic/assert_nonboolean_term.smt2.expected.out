(error "Top-level assertion sort must be Bool, got Real")

